src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -94,6 +96,21 @@ export class Message extends React.Component<Props, any> {
     })
   }
 
+  renderBody() {
+    const { message, firstMessage, initialText } = this.props
+    const isSent = !!message.created_at
+    let body = firstMessage ? initialText : message.body
+
+    // TODO: When we save purchase request initial text properly, this hack can be removed.
```

Comment:
Unfortunately, our grand plan (which works perfectly!), is stymied by purchase requests: https://github.com/artsy/impulse/issues/349

Essentially when we handle that better in the above issue, we can just kill this `split` and hacky thing.

Reply:
I'm for leaving it for now, since at its worst, it will operate as if it didn't do the check, and it _will_ fix some.

Reply:
Actually, I'm breaking the cardinal rule! I'll move this to Metaphysics.